modules/raft/src/main/java/org/apache/ignite/raft/jraft/option/NodeOptions.java

```diff
@@ -301,6 +303,14 @@ public class NodeOptions extends RpcOptions implements Copiable<NodeOptions> {
      */
     private boolean isSystemGroup = false;
 
+    /**
+     * Shared pool of {@link ByteBufferCollector} for sending log entries for replication.
+     *
+     * <p>Used to prevent a large number of {@link ByteBufferCollector} from being accumulated across all threads that are involved in
+     * sending log entries, see {@link ByteBufferCollector#allocateByRecyclers}.</p>
+     */
+    private ByteBufferCollectorPool appendEntriesByteBufferCollectorPool;
```

Comment:
It seems that in a few places in our code, the same implementation is assigned for this option. Would it make sense to initialize the field with it right here? In this way we'll make it more difficult to have different pools in different tests, for example